server/app/api/v3.py

```diff
@@ -182,9 +182,9 @@ def add_resource(resource, url):
     add_resource(AnnotationsObsAPI, "/annotations/obs")
     add_resource(AnnotationsVarAPI, "/annotations/var")
     add_resource(DataVarAPI, "/data/var")
+    add_resource(GeneInfoAPI, "/geneinfo")
```

Comment:
did changing the order do anything? If not lets revert this change

Reply:
I went ahead and changed the order after you pointed out that the routes are organized in alphabetical order, but I can move it back if you think that doesn't matter?